src/main/java/io/supertokens/webserver/api/multitenancy/BaseCreateOrUpdate.java

```diff
@@ -138,6 +139,8 @@ protected void handle(HttpServletRequest req, TenantIdentifier sourceTenantIdent
             throw new ServletException(new BadRequestException("Invalid core config: " + e.getMessage()));
         } catch (InvalidProviderConfigException e) {
             throw new ServletException(new BadRequestException("Invalid third party config: " + e.getMessage()));
+        } catch (InvalidFormatException e) {
```

Comment:
this should ideally be a part InvalidConfigException with a message like "Cannot deserialize value abcd for property access_token_validity. This field only accepts type: long"